charts/edc-bundle/Chart.yaml

```diff
@@ -0,0 +1,46 @@
+# #############################################################################
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+apiVersion: v2
+name: edc-bundle
+description: |
+  EDC Bundle Chart for data transfer capabilities in Catena-X.
+  This bundle includes:
+  - Eclipse Dataspace Connector (EDC) v0.9.0
```

Comment:
In order to avoid mismatch between configuration and description do not use static version string here